pkg/sql/privilege/privilege.go

```diff
@@ -31,34 +33,60 @@ type Kind uint32
 // Do not change values of privileges. These correspond to the position
 // of the privilege in a bit field and are expected to stay constant.
 const (
-	ALL    Kind = 1
```

Comment:
We did this very intentionally. There was a whole kerfuffle when a new entry got added into the middle of the list. I think when things are durable it makes sense to not use iota.

Reply:
I think the test I added should prevent people from adding or removing values from the middle (replacing a value with `_` should be fine).